geographer/utils/utils.py

```diff
@@ -169,12 +172,19 @@ def concat_gdfs(objs: list[GeoDataFrame], **kwargs: Any) -> GeoDataFrame:
     list.
     """
     for obj in objs:
-        if isinstance(obj, GeoDataFrame) and obj.crs != objs[0].crs:
-            raise ValueError("all geodataframes should have the same crs")
+        if isinstance(obj, GeoDataFrame):
+            if obj.crs != objs[0].crs:
+                raise ValueError("All geodataframes should have the same crs!")
```

Comment:
I know this is _totally outrageous_, but maybe no exclamation mark is needed?

```suggestion
                raise ValueError("All geodataframes should have the same CRS")
```

Reply:
Totally!!!

Reply:
Fixed!!!!